docs/02-app/02-api-reference/05-next-config-js/reactStrictMode.mdx

```diff
@@ -15,4 +15,6 @@ module.exports = {
 }
 ```
 
+> **Good to know**: Since Next.js 13.4, Strict Mode is `true` by default, so the above configuration is not necessary. You can still disable Strict Mode by setting `reactStrictMode: false`.
```

Comment:
```suggestion
> **Good to know**: Since Next.js 13.4, Strict Mode is `true` by default with `app` router, so the above configuration is only necessary for `pages`. You can still disable Strict Mode by setting `reactStrictMode: false`.
```